doc/client_lua_api.md

```diff
@@ -774,6 +774,37 @@ An interface to manipulate minimap on client UI
 * `get_mode()`: returns the current minimap mode
 * `toggle_shape()`: toggles minimap shape to round or square.
 
+### LocalPlayer
+An interface to retrieve information about the player.
+
+* `get_velocity()`: returns player send
+* `get_hp()`: returns player HP
+* `got_teleported()`: 
+* `is_attached()`:
+* `is_touching_ground()`:
+* `is_in_liquid()`:
+* `is_in_liquid_stable()`:
+* `get_liquid_viscosity()`:
+* `is_climbing()`:
+* `swimming_vertical()`:
+* `get_physics_override()`:
+* `get_override_position()`:
+* `get_last_position()`:
+* `get_last_speed()`:
+* `get_last_pitch()`:
+* `get_last_yam()`:
```

Comment:
If you call it `get_last_horizontal` (or `get_last_look_horizontal`), the `get_last_pitch()` also has to be changed to `get_last_vertical` (or `get_last_look_vertical`). If you leave the `get_last_pitch()` as it is, call `get_last_yam()` `get_last_yaw()`.